source/configuration.txt

```diff
@@ -234,9 +255,8 @@ Partitioner Configuration
 
 .. note::
 
-   If setting these connector configurations via ``SparkConf``, prefix
-   these configuration settings with
-   ``spark.mongodb.input.partitionerOptions.``.
+   If you are using ``SparkConf``, prefix
+   ``spark.mongodb.input.partitionerOptions.`` to each property.
```

Comment:
Suggestion: Since this is used so many times, I think we should make it an include. That way, if this needs to change again in the future, we only need to edit one place.